tests/signal-cast-registerer.cpp

```diff
@@ -139,15 +139,6 @@ BOOST_AUTO_TEST_CASE(standard_double_registerer) {
 BOOST_AUTO_TEST_CASE(custom_vector_registerer) {
   dynamicgraph::Signal<dynamicgraph::Vector, int> myVectorSignal("vector");
 
-  /// Create a second local vector registerer to generate an exception.
-  bool res = false;
-  try {
-    EigenCastRegisterer_V myVectorCast2;
-  } catch (const ExceptionSignal &aes) {
-    res = (aes.getCode() == ExceptionSignal::GENERIC);
-  }
-  BOOST_CHECK(res); // Uncomment this line to resolve the unused-but-set-variable warning to have a stable release in ros buildfarm
-
```

Comment:
Dear @Rascof thanks for the fix.
For the record the point of those lines was to trigger the exception.
It seems that this is not the case here.
Thanks for removing it.